bin/ldaptools

```diff
@@ -23,9 +23,10 @@ when 'group'
   commands('This object is used for group management', cmd, GROUP_SUB_COMMANDS)
   Tapjoy::LDAP::Group.new
 when 'key'
-  KEY_SUB_COMMANDS = %w(add remove install)
+  KEY_SUB_COMMANDS = %w(add remove install list)
```

Comment:
Is the intent here that you run ldaptools list <username> and get a key back? If so I think 'show' is better. List implies to me that you're getting all of something.